inference-engine/src/vpu/myriad_plugin/myriad_plugin.cpp

```diff
@@ -49,10 +50,6 @@ ExecutableNetworkInternal::Ptr Engine::LoadExeNetworkImpl(
         manager.register_pass<vpu::DynamicToStaticShape>();
         manager.register_pass<vpu::EliminateShapeOfAfterDSR>();
         manager.run_passes(function);
-
-        ngraph::pass::Manager ti_manager;
-        ti_manager.register_pass<ngraph::pass::ApplyTransformationsToTIBody>(manager);
-        ti_manager.run_passes(function);
```

Comment:
Should the same be done for HDDL and FrontEnd of myriad plugin?

@AndrewBakalinIntel @Maxim-Doronin please take a look

Reply:
This pass was recently introduced, so I believe that other plugins do not use it.

Reply:
That's not true: https://github.com/openvinotoolkit/openvino/blob/master/inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp#L411

Could you please align transformation for HDDL plugin as well?

Reply:
This PR already have this changes: https://github.com/openvinotoolkit/openvino/pull/2114/files#diff-a0299d085966b3f0193f4d2adc2b7e8cL26